src/App.js

```diff
@@ -1,29 +1,27 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Route, Routes } from 'react-router-dom';
+import { useDispatch, Provider } from 'react-redux';
+import Home from './pages/Home';
+import StockDetails from './pages/StockDetails';
+import NotMatch from './pages/NotMatch';
+import { getStocks } from './redux/stocks/stocks';
+import store from './redux/configureStore';
 import './App.css';
 
-function App() {
+const App = () => {
+  const dispatch = useDispatch();
+  dispatch(getStocks());
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Provider store={store}>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/home" element={<Home />} />
+        <Route path="/stock-details/:id" element={<StockDetails />} />
```

Comment:
- [ ] Your unique routing does not work. When I click on any item on the home page it takes me to this route `/stock-details`, instead of this `/stock-details/:id`.